src/app/app.scss

```diff
@@ -1,3 +1,10 @@
+// Variables
+@import './styles/variables/variables.css';
```

Comment:
why do you import .css files in .scss?

Reply:
Files in imports changed from .css to .scss.